visqol/Cargo.toml

```diff
@@ -19,7 +19,7 @@ example = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-visqol-rs = "0.1.1"
+visqol-rs = { path = "../visqol-rs" }
```

Comment:
I would prefer to keep the binary decoupled from the library. Could you revert this change?

Reply:
Sure!